js/TutorModel.js

```diff
@@ -12,6 +12,7 @@ export default class TutorModel extends Backbone.Model {
         text: '{{_globals._extensions._tutor.hideFeedback}}',
         ariaLabel: '{{_globals._extensions._tutor.hideFeedback}}'
       },
+      altTitle:'{{_globals._extensions._tutor.altTitle}}',
```

Comment:
This value will always be replaced by the one in course.json:_tutor.altTitle on the next line `...Adapt.course.get('_tutor')` when it exists on the course model, which it will in the schema and the example.json

Reply:
This is probably more what you meant:

```js
   const defaults = {
      _type: TUTOR_TYPE.NOTIFY.asLowerCase,
      _classes: '',
      _hasNotifyBottomButton: false,
      _button: {
        text: '{{_globals._extensions._tutor.hideFeedback}}',
        ariaLabel: '{{_globals._extensions._tutor.hideFeedback}}'
      },
      ...Adapt.course.get('_tutor')
    };
    defaults.altTitle ??= Adapt.course.get('_globals')._extensions._tutor.altTitle;
    return defaults
```
Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment

It's not a good place to put this default as the question will need to pass through its own alternate questionFeedback title, which when explicitly set to a null string, should output nothing here. There is no other instance of a tutor showing except when triggered from a question feedback.